app/views/notifier/review_orders.text.haml

```diff
@@ -0,0 +1,8 @@
+= text "body_intro", facility: @facility, smart: false
+
+- @accounts.each do |account|
```

Comment:
Not sure what's doing it (probably HAML), but there's no empty line between the intro and the list of accounts.